### PR TITLE
AddBookActivityToFragment

### PR DESCRIPTION
**Pull request by ghilbing**
Add Book Fragment basic

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -58,6 +59,5 @@ dependencies {
     annotationProcessor 'com.github.hotchemi:permissionsdispatcher-processor:2.4.0'
     compile 'com.github.bumptech.glide:glide:4.1.1'
```
Comment (m3libea) [inline app/build.gradle]: this should be glide:2.3.3 in order to work with the stable version

File: app/src/main/java/com/fantastic/bookxchange/activities/UserActivity.java [hunk 2]
```diff
@@ -99,6 +105,15 @@ private void setupTab() {
     @Override
     public void onClickListener(Book book) {
         //TODO Itent to Book Detail page
+        Log.i("Book", book.getTitle().toString());
+        Toast.makeText(this, book.getTitle().toString(), Toast.LENGTH_LONG).show();
```
Comment (m3libea) [inline app/src/main/java/com/fantastic/bookxchange/activities/UserActivity.java]: I think this is also on the other PR, please review :)

File: app/src/main/java/com/fantastic/bookxchange/adapters/BooksAdapter.java [hunk 3]
```diff
@@ -78,6 +82,12 @@ public void onBindViewHolder(ViewHolder holder, int position) {
         holder.llBook.setOnClickListener(view -> {
             rowSelectedIndex=position;
             listener.onBookClickListener(book);
+            // call Detail fragment
+            Intent intent = new Intent(context, BookDetailActivity.class);
```
Comment (m3libea) [inline app/src/main/java/com/fantastic/bookxchange/adapters/BooksAdapter.java]: Same thing, also on the other PR

**Comment by ghilbing** [comment]
Another PR replace this one

**Pull request closed without merging**